libraries/cloudharness-common/cloudharness/utils/env.py

```diff
@@ -79,21 +89,21 @@ def name_to_variable(application_name):
 
 # CloudHarness Events
 def get_cloudharness_events_client_id():
-    return get_variable('CH_KEYCLOAK_WEBCLIENT_ID')
+    return 'web-client'
```

Comment:
The same value is available from `.Values.apps.accounts.webclient.id`

Reply:
yes agree